.circleci/config.yml

```diff
@@ -1,21 +1,16 @@
 version: 2.1 # use CircleCI 2.1
```

Comment:
We can remove the changes I made in this file.

1. They should be done in a different PR (with node14/16 added as well).
2. I think I only did those because I thought (mistakenly) there was some issue with old node10